Examples/topics/Sources/main.swift

```diff
@@ -17,77 +17,76 @@ func main() async {
 
     let subscribeResponse = await client.subscribe(cacheName: cacheName, topicName: topicName)
     
+    let subscription: TopicSubscription?
```

Comment:
It actually looks like you can drop the `?` here and the `!` below from `for try await item in subscription!.stream`. Looks like the compiler is smart enough to know that it will get a value (which is also why you can declare it with `let` instead of `var` . . . TIL!).